x-pack/plugins/aiops/public/components/document_count_content/document_count_chart/document_count_chart.tsx

```diff
@@ -336,52 +336,56 @@ export const DocumentCountChart: FC<DocumentCountChartProps> = ({
             theme={chartTheme}
             baseTheme={chartBaseTheme}
             debugState={window._echDebugStateFlag ?? false}
+            showLegend={false}
+            showLegendExtra={false}
           />
+          <Axis id="aiops-histogram-left-axis" position={Position.Left} ticks={2} integersOnly />
           <Axis
-            id="bottom"
+            id="aiops-histogram-bottom-axis"
             position={Position.Bottom}
             showOverlappingTicks={true}
             tickFormat={(value) => xAxisFormatter.convert(value)}
+            // temporary fix to reduce horizontal chart margin until fixed in Elastic Charts itself
+            labelFormat={() => ''}
             timeAxisLayerCount={useLegacyTimeAxis ? 0 : 2}
             style={useLegacyTimeAxis ? {} : MULTILAYER_TIME_AXIS_STYLE}
```

Comment:
This workaround works only with the multilayer time axis and not with the legacy.
So my suggestion is to remove completely the legacyTimeAxis configuration and use only the multilayer one.
If you prefer keep that, you instead should change the labelFormat as:
```tsx
labelFormat={useLegacyTimeAxis ? undefined : () => ''}
```
you can double check how it will work by setting the legacy time axis in the Advanced setting

Reply:
Thanks for the hint! I'll stick with `useLegacyTimeAxis` for now so the chart behavior is the same as in Discover.